noir_stdlib/src/scalar_mul.nr

```diff
@@ -1,2 +1,8 @@
+// Computes a fixed base scalar multiplication over the embedded curve.
+// For bn254, We have Grumpkin and Baby JubJub.
+// For bls12-381, we have JubJub and Bandersnatch.
+//
+// The embedded curve being used, currently is decided by the 
```

Comment:
```suggestion
// The embedded curve being used is decided by the 
```
Nit: This didn't read well to me with the comma and "currently". Any comments are implicitly what is currently supported as well. If it is expected to change in the future, we can make that a separate note. However, this is non blocking for merging imo.